Index: maze_solver_AI/src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Hello world!\");\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/maze_solver_AI/src/Main.java b/maze_solver_AI/src/Main.java
--- a/maze_solver_AI/src/Main.java	(revision c04282c08898389671b90445162571e12e17ea84)
+++ b/maze_solver_AI/src/Main.java	(date 1678180540324)
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("a");
     }
 }
\ No newline at end of file
Index: maze_solver_AI/src/Grid.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/maze_solver_AI/src/Grid.java b/maze_solver_AI/src/Grid.java
new file mode 100644
--- /dev/null	(date 1678180759207)
+++ b/maze_solver_AI/src/Grid.java	(date 1678180759207)
@@ -0,0 +1,10 @@
+public class Grid {
+    int size;
+    public Grid(){
+        size=10;
+    }
+
+    public Grid(int size){
+        this.size=size;
+    }
+}
Index: maze_solver_AI/src/App.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/maze_solver_AI/src/App.java b/maze_solver_AI/src/App.java
new file mode 100644
--- /dev/null	(date 1678180953514)
+++ b/maze_solver_AI/src/App.java	(date 1678180953514)
@@ -0,0 +1,6 @@
+public class App {
+
+    public App(int time,int steps){
+        System.out.println("Time:"+time+" Steps:"+steps);
+    }
+}
